src/main/java/seedu/address/model/person/QualificationContainsQualificationPredicate.java

```diff
@@ -0,0 +1,33 @@
+package seedu.address.model.person;
+
+import java.util.List;
+import java.util.function.Predicate;
+
+/**
+ * Tests that a {@code Person}'s {@code Qaulification} matches the qualification given.
+ */
+public class QualificationContainsQualificationPredicate implements Predicate<Person> {
```

Comment:
Perhaps this class name is too long, but I'm not sure if there's a better naming system for it anyway😟